tests/test_utils.cpp

```diff
@@ -60,87 +60,104 @@ void printIssues(const libcellml::LoggerPtr &l, bool headings, bool causes, bool
     }
 }
 
-void printModel(const libcellml::ModelPtr &model)
-{
-    std::cout << "The model name is: '" << model->name() << "'" << std::endl;
-    if (model->id() != "") {
-        std::cout << "The model id is: '" << model->id() << "'" << std::endl;
-    }
-
-    // 2.a    Print any custom units of the model
-    std::cout << "The model defines " << model->unitsCount()
-              << " custom units:" << std::endl;
-    for (size_t u = 0; u < model->unitsCount(); ++u) {
-        std::cout << "  Units[" << u << "] is '" << model->units(u)->name() << "'"
-                  << std::endl;
-    }
+static const std::string INDENT = "    ";
```

Comment:
To reduce confusion with later uses of `indent` I think this variable should be renamed to `FIXED_INDENT`.

Reply:
f0f249dc